config/common/temporal.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+use Temporal\DataConverter\DataConverter;
+use Temporal\DataConverter\DataConverterInterface;
+use Temporal\Worker\Transport\Goridge;
+use Temporal\Worker\Transport\RPCConnectionInterface;
+use Temporal\Worker\WorkerOptions;
+use Temporal\WorkerFactory;
+
+/**
+ * @var $params array
+ */
+
+if (!($params['yiisoft/yii-runner-roadrunner']['temporal']['enabled'] ?? false)) {
+    return [];
+}
+
+$options = $params['yiisoft/yii-runner-roadrunner']['temporal']['options'] ?? [];
+
+
+return [
+    'tag@temporal.workflow' => [],
+    'tag@temporal.activity' => [],
+
+    WorkerOptions::class => [
+        //        'withMaxConcurrentActivityExecutionSize()' => [$options['maxConcurrentActivityExecutionSize']],
```

Comment:
I don't know how to configure these parameters properly and I've decided to leave it here just for examples